src/cdev/project_templates/packages/src/__init__.py

```diff
@@ -1 +1,2 @@
+# ANIBAL IS this intended?
```

Comment:
No this can be removed. I'll look into adding a pre-commit linter for `print` statements in the codebase